src/picai_prep/dcm2mha.py

```diff
@@ -520,9 +524,45 @@ def _read_image(self, path: Optional[PathLike] = None) -> sitk.Image:
         try:
             return self._read_image_sitk(path=path)
         except RuntimeError:
-            return self._read_image_pydicom(path=path)
+            # try again while removing localizer slices
```

Comment:
please explain why localizer slices might cause a RuntimeError?

Reply:
When the slices in an folder do not form a 3D volume, then SimpleITK throws an `RuntimeError`. This happens e.g. when one/multiple localizer slices are added to a normal scan's folder, or when the folder contains only localizer slices.